autoload/neomake/makers/ft/php.vim

```diff
@@ -42,8 +42,19 @@ function! neomake#makers#ft#php#phpmd() abort
 endfunction
 
 function! neomake#makers#ft#php#phpstan() abort
-    return {
-        \ 'args': ['analyse', '--errorFormat', 'raw'],
+    " PHPStan normally considers 0 to be the default level, so that is used here as the default:
+    let maker = {
+        \ 'args': ['analyse', '--errorFormat', 'raw', '--level', get(g:, 'neomake_phpstan_level', 0)],
         \ 'errorformat': '%E%f:%l:%m',
         \ }
+    " Check for the existence of a default PHPStan project configuration file.
+    " Technically PHPStan does not have a concept of a default filename for a
+    " project configuration file, but phpstan.neon is the filename shown in the
+    " example in the PHPStan documentation, so this is the default name expected
+    " by Neomake.
+    let phpStanConfigFilePath = neomake#utils#FindGlobFile('phpstan.neon')
+    if !empty(phpStanConfigFilePath)
+        call extend(maker.args, ['-c', phpStanConfigFilePath])
+    endif
+    return maker
```

Comment:
I assume the config would override the `--level` in this case?
It might be easier to not support a custom `--level`, but only the config (if the level can be configured there) - or otherwise append `--level` at the end always (assuming that it overrides the config then)?!
But maybe `--level` overrides `-c` always already?